tests/console/check_system_info.pm

```diff
@@ -189,7 +190,9 @@ sub run {
 
     # feature SLE-21783
     # Check nodejs-common on SLES15SP3+ before and after migration
-    if (get_var('SCC_ADDONS') =~ /wsm/ && is_sle('>=15-SP3') && !get_var('MEDIA_UPGRADE')) {
+    # We just check nodejs_common for sle15-sp3+ and wsm module was registered
+    my $ret = script_run("SUSEConnect --status-text|grep -A3 -E 'Web and Scripting Module' | grep -qE '^\\s+Registered'");
```

Comment:
Why need check the wsm is registered or not? If wsm is not registered, is it a bug or test issue?

Reply:
in zdup we disable sle at that point. so check if ws was registered to avoid all conditions for media and zdup alike.

Reply:
I think it is better record the info that wsm is not registered, so nodejs hasn't been tested.